catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -203,8 +342,7 @@
                             guava,
                             notifications,
                             activities,
```

Comment:
Yeah, that's what I figured after asking the question. Thanks for confirming what I thought :+1: